### PR TITLE
TVAULT-3187 Shell changes reflect in ansible

### PR DESCRIPTION
**Pull request by yogeshrathoret15**
Above changes tested on Redhat based setup only
Currently ubuntu based system using in sanity and testing
once free will test changes on ubuntu based system also

### CODE REVIEW

File: ansible/roles/ansible-horizon-plugin/tasks/remove_trilio_pth.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  shell: "{{ PYTHON_VERSION }} -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())'"
+  register: TRILIOFILE
+
+- set_fact:
+    TRILIO_FILE : "{{TRILIOFILE.stdout}}/trilio.pth"
+
+- name: Removing trilio.pth file
+  file:
+    path: "{{ TRILIO_FILE }}"
+    state: absent
```
Comment (abhijeetpatra) [inline ansible/roles/ansible-horizon-plugin/tasks/remove_trilio_pth.yml]: Rename TRILIOFILE with SITEPKG_LOC

File: ansible/roles/ansible-tvault-contego-extension/tasks/get_venv.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    RECOMMENDS_VAR : "no"
+  when: ENV_PATH != "/usr"
+
+- set_fact:
+    RECOMMENDS_VAR : "yes"
+  when: ENV_PATH == "/usr"
+
+- name: Found location of trilio.pth file
+  shell: "{{ PYTHON_VERSION }} -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())'"
+  register: TRILIOFILE
```
Comment (abhijeetpatra) [inline ansible/roles/ansible-tvault-contego-extension/tasks/get_venv.yml]: Rename TRILIOFILE to VENV_PKGLIB

File: ansible/roles/ansible-tvault-contego-extension/templates/s3_others.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+vault_data_directory = {{TVAULT_DATA_DIR}}
+log_file = /var/log/nova/tvault-contego.log
+debug = False
+verbose = True
+max_uploads_pending = 3
+max_commit_pending = 3
+vault_s3_auth_version = DEFAULT
+vault_s3_access_key_id = {{VAULT_S3_ACCESS_KEY}}
+vault_s3_secret_access_key = {{VAULT_S3_SECRET_ACCESS_KEY}}
+vault_s3_region_name = us-east-1
```
Comment (abhijeetpatra) [inline ansible/roles/ansible-tvault-contego-extension/templates/s3_others.conf.j2]: Need to have a variable for region name as well.

Comment (yogeshrathoret15) [reply ansible/roles/ansible-tvault-contego-extension/templates/s3_others.conf.j2]: Requested changes done

File: ansible/roles/ansible-tvault-contego-extension/tasks/get_venv.yml (hunk 2 above)
Comment (yogeshrathoret15) [reply ansible/roles/ansible-tvault-contego-extension/tasks/get_venv.yml]: Requested changes done

File: ansible/roles/ansible-horizon-plugin/tasks/remove_trilio_pth.yml (hunk 1 above)
Comment (yogeshrathoret15) [reply ansible/roles/ansible-horizon-plugin/tasks/remove_trilio_pth.yml]: Requested changes done

**Pull request merged**